src/app/ram_fix_math/ram_fix_math.ml

```diff
@@ -0,0 +1,812 @@
+open Core
+
+(* number of operations to do when performing benchmarks *)
+let bench_count = 10_000
+
+module Const = struct
+  let k = 290
+
+  let l = 60
```

Comment:
@nholland94 what is `l` ? The amount of estimated blocks and scan states in the frontier are now depend on this value, what changed?

Reply:
This was just committed by accident. I was computing something locally real quick.